routes/v1.js

```diff
@@ -766,6 +772,7 @@ api.declare({
   method:     'post',
   route:      '/task/:taskId/rerun',
   name:       'rerunTask',
+  stability:  base.API.stability.experimental,
```

Comment:
I don't want people to call it... are you sure wish to keep it..

It sort of breaks the concept that a task only has one resolution... If rerunTask is called a task can have multiple pulse messages saying it's been resolved in different ways... That can't happen if rerunTask is removed...

Yes, we could still have multiple pulse messages, but they would just be duplicates... Not announcing different resolutions..

IMO, people shouldn't care about "runs".... But they do have to care about them now, because we can two runs that causes a resolution, if rerunTask is called.